src/cow-react/modules/limitOrders/updaters/QuoteUpdater/hooks/useQuoteRequestParams.ts

```diff
@@ -17,7 +17,7 @@ export function useQuoteRequestParams(): FeeQuoteParams | null {
   const feeQuoteParams: FeeQuoteParams | null = isFullInput
     ? {
         chainId,
-        validTo: Math.round(Date.now() / 1000 + 60 * deadline),
+        validTo: Math.round((Date.now() + deadline) / 1000),
```

Comment:
Will this work also with the "custom" date?

Reply:
Yes, we align it in the `useLimitOrdersTradeState()`